### ISSUE TITLE
struggling to correlate some RHOC accounts to REV accounts

### ISSUE THREAD

**Issue by dckc** [rchain/rbalance#10]
I'm auditing with [TEST.rst](https://github.com/dckc/rbalance/blob/check_rev/TEST.rst). I can pretty much account for the adjustments to RHOC accounts, but I'm struggling to see, for example, where the **Reserve Wallet 0x1c73d** ended up. The proposed [wallets.txt](https://github.com/rchain/rchain/blob/dev/wallets.txt) has some addresses that were not in the RHOC snapshot. The total is 1B (modulo a bit of dust; see #7) so the overall effect seems plausible, but I'm struggling to correlate exactly which new address goes with which old address.

```
File "TEST.rst", line 326, in TEST.rst
Failed example:
    audit.show('{0:<44} {1:>20} {2:>20} {3:>20}', decimals=8, *db.query('''
      select coalesce(coalesce(bk.label, t.label) || ' ' || substr(adj.addr, 1, 7), adj.addr) addr
           , adj.bal_rhoc, adj.bal_rev, adj.delta from adj
      left join addrbook bk on bk.addr = adj.addr
      left join (select 'feb 11 taint' label, t.* from taint t) t on t.addr = adj.addr
      order by abs(delta) desc, addr
    '''))
Expected:
    'raw RHOC to REV adjustments with addresses labelled... not sure what they should be@@'
Got:
    addr                                                     bal_rhoc              bal_rev                delta
    Reserve Wallet 0x1c73d                         274664038.37716800                 0E-8  -274664038.37716800
    0x6defba912a6664838eec10417c75d5270932d6c7                          262347004.13716800   262347004.13716800
    0xc9b2b0bbc1558d69fd285d31ee7897d9b808103a                          141993476.15949200   141993476.15949200
    Current operation 0xd35a2                       77932217.80308682                 0E-8   -77932217.80308682
    Token Sale Wallet 0x28755                       31176931.98495265                 0E-8   -31176931.98495265
    0xb4c242f379eed1f2a6cdbc1ca7466738f06793a5                           31176931.98495260    31176931.98495260
    pithia 3 19,499,000 0x62917                     21088740.00000000     1588740.00000000   -19500000.00000000
    pithia 4 1,000 0x62917                          21088740.00000000     1588740.00000000   -19500000.00000000
    feb 11 taint 0x583c3                            14838673.35645073                 0E-8   -14838673.35645073
    new REV 0xf1523                                 14208810.00000000                 0E-8   -14208810.00000000
    pithia 7 8,936,500 0xaa9bd                       8927500.00000000                 0E-8    -8927500.00000000
    pithia 8 1,000 0xaa9bd                           8927500.00000000                 0E-8    -8927500.00000000
    0x42c9625ea0b18a6d427048094a14476cf339cd31                            7104405.00000000     7104405.00000000
    0xeb08e33fdbb693ac2fddd104de0b8a2ac56d6119                            7104405.00000000     7104405.00000000
    pithia 10 1,000 0x44d37                          6466991.46410000      216991.46410000    -6250000.00000000
    pithia 9 6,466,991 0x44d37                       6466991.46410000      216991.46410000    -6250000.00000000
    KuCoin 2 0x689c5                                28826642.88573629    23826642.88573620    -5000000.00000009
    RHOC Scam token 14 0x3198a                       4315002.00000000                 0E-8    -4315002.00000000
    pithia 5 4,061,500 0xbdcbf                       5122041.08255400     1059541.08260000    -4062499.99995400
    pithia 6 1,000 0xbdcbf                           5122041.08255400     1059541.08260000    -4062499.99995400
    Research Wallet 0x821aa                          4000000.00000000                 0E-8    -4000000.00000000
    0x742ab73a29239d0bbdb8548d936f1325a58dd8fb                            3872000.00000000     3872000.00000000
    RHOC Scam token 2 0xdcb05                        3400500.00000000                 0E-8    -3400500.00000000
    0x198eddaac45d28ec336df3443dfc5f23d16a8a52                            3000000.00000000     3000000.00000000
    0x17e31523fe2643b8da359c0a1c9c17c200f40cb6                            1500000.00000000     1500000.00000000
    0x2f345a89e5a8b6911aa474693489b5abd3becd26                            1500000.00000000     1500000.00000000
    0x7e873b3aef3e68cd96597b60ca51945c0e1e9260                            1500000.00000000     1500000.00000000
    0x9bd50d19d52fce03dca7f0acebf709902c273b06                            1500000.00000000     1500000.00000000
    0xcd98d2ee701a57b804bccf96b985fb54a825aa82                            1500000.00000000     1500000.00000000
    0x1beb2a2094cb11175bf289fb4f65698c41ca2afc                            1000000.00000000     1000000.00000000
    0x2f5c1c8555a4b679e11ada58959fdc5f6f79cf4c                            1000000.00000000     1000000.00000000
    0xaca549e1142a46c9e329008879e9a5389e8985f3                            1000000.00000000     1000000.00000000
    0xc480fc9a741e6e4eb7a9d701bdb6e53f923e8583                            1000000.00000000     1000000.00000000
    0x5333e2064df92d85321ffdab03620f44481442b8                             987444.54099958      987444.54099958
    0x66433026f44e69bf2695f05de3d4e52aa2268198                             817034.24000000      817034.24000000
    new REV 0x4c8c0                                   783513.78500000                 0E-8     -783513.78500000
    feb 11 taint 0xfd9b2                              364784.00000000                 0E-8     -364784.00000000
    feb 11 taint 0x5c13a                              300000.00000000                 0E-8     -300000.00000000
    new REV 0xc3a0f                                   203930.75599958                 0E-8     -203930.75599958
    feb 11 taint 0x44948                              135299.00000000                 0E-8     -135299.00000000
    0xf6e07f4ae0961a143d164585fc5f134ec438a1ca                             128000.00000000      128000.00000000
    0x95a110f5fcc7cec23e072840f15450817e5e6c90                              48000.00000000       48000.00000000
    feb 11 taint 0x6e75b                               10000.00000000                 0E-8      -10000.00000000
    RHOC Scam token 17 0xbbd93                          5000.00000000                 0E-8       -5000.00000000
```

**Comment by dckc** [comment]
I'm still not sure where some of these come from; in particular:
https://github.com/dckc/rbalance/commit/4d2554977703057882cca4c2acd61abe945829e9

Where did the **135767221.20220800** REV in `0xc9b2b0bbc1558d69fd285d31ee7897d9b808103a` come from? Where did the **77932217.80308682** RHOC in **Current operation 0xd35a2** go to? and so on.

**Comment by dckc** [comment]
@ian-bloom helped me see what goes where.

fixed in a306fc9 https://github.com/dckc/rbalance/blob/check_rev/TEST.rst
